File: java/src/br/inatel/pos/mobile/dm102/projeto/model/PessoaJuridica.java
package br.inatel.pos.mobile.dm102.projeto.model;

public class PessoaJuridica extends Cliente {
	
	private String cnpj;
	private String razaoSocial;
	private String inscricaoEstadual;
	
	public PessoaJuridica(Cliente cliente, String cnpj,
			String razaoSocial, String inscricaoSocial) {
		super(cliente.getNome(), cliente.getEndereco(), cliente.getTelefone());
		this.cnpj = cnpj;
		this.razaoSocial = razaoSocial;
		this.inscricaoEstadual = inscricaoSocial;
	}
	
	public String getCnpj() {
		return cnpj;
	}

	public String getRazaoSocial() {
		return razaoSocial;
	}

	public String getInscricaoEstadual() {
		return inscricaoEstadual;
	}

	@Override
	public String toString() {
		return "[Pessoa Jurídica] " + super.toString() + " CNPJ: " + cnpj + " Razão Social: " + razaoSocial + " Inscrição estadual: " + inscricaoEstadual;
	}
	
	
}

File: c#/ProjetoDM102/PessoaJuridica.cs
using System;
namespace ProjetoDM102
{
    public class PessoaJuridica : Cliente
    {
        public String cnpj { get; }
		private String razaoSocial;
		private String inscricaoEstadual;

        public PessoaJuridica(string nome,string endereco, string telefone,
                              string cnpj, string razaoSocial, string inscricaoEstadual) : base (nome, endereco, telefone)
        {
            this.cnpj = cnpj;
            this.razaoSocial = razaoSocial;
            this.inscricaoEstadual = inscricaoEstadual;
        }

		public override string ToString()
		{
			return "[Pessoa Jurídica] " + base.ToString() + " CNPJ: " + cnpj + " Razão Social: " + razaoSocial + " Inscrição estadual: " + inscricaoEstadual;
		}
    }
}
